### ISSUE TITLE
Update to 2.1.8, admin side menu disapears

### ISSUE THREAD

**Issue by tobasium** [simatec/ioBroker.backitup#490]
After Update to 2.1.8 the admin5 side menu disapears only logout will be shown. 

Admin
v5.0.24
Node.js
v12.22.1
NPM
6.14.12

**Comment by Black-Thunder** [comment]
Same here. This appears when trying to open the "Backup" tab:

![image](https://user-images.githubusercontent.com/972581/118360377-874b3f00-b587-11eb-9d5f-f667ac2d418e.png)

Dev console output:

```
iobroker.admin@5.0.24
instrument.ts:129 [2021-05-15T12:09:56.325Z] Try to connect
Drawer.js:414 Uncaught (in promise) TypeError: Cannot set property 'visible' of undefined
    at Drawer.js:414
    at Array.map (<anonymous>)
    at Drawer.js:412
```

**Comment by simatec** [comment]
Please Test 2.1.9 with admin 5.0.25

**Comment by Black-Thunder** [comment]
After updating both adapters the side menu is fixed for me. But trying to open the "Backup" tab (/#tab-backitup) still shows the same error.

![image](https://user-images.githubusercontent.com/972581/118392861-0bf79500-b63c-11eb-98bd-feb89fdd26cb.png)

 This time there is no more error message in the dev console.

```
iobroker.admin@5.0.25
instrument.ts:129 [2021-05-16T09:47:04.380Z] Try to connect
fetch.ts:126 Fetch finished loading: POST "https://sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7".
socket.io.js:59 [2021-05-16T09:47:06.866Z] Try to connect
translate.js:24 "Backitup": {"en": "Backitup", "de": "Backitup", "ru": "Backitup", "pt": "Backitup", "nl": "Backitup", "fr": "Backitup", "es": "Backitup", "pl": "Backitup", "it": "Backitup", "zh-cn": "Backitup"},
translate.js:24 "ioBroker": {"en": "ioBroker", "de": "ioBroker", "ru": "ioBroker", "pt": "ioBroker", "nl": "ioBroker", "fr": "ioBroker", "es": "ioBroker", "pl": "ioBroker", "it": "ioBroker", "zh-cn": "ioBroker"},
```

**Comment by Black-Thunder** [comment]
Updated the adapter to 2.1.10 and the issue is now fixed for me. Thanks for the quick update!

**Comment by simatec** [comment]
Fix in 2.1.10
